main.go

```diff
@@ -97,41 +103,81 @@ func main() {
 	log.Infof("Release created:")
 	log.Printf(newRelease.GetHTMLURL())
 
-	if filelist := strings.TrimSpace(c.FilesToUpload); filelist != "" {
-		fmt.Println()
-		log.Infof("Uploading assets:")
+	filelist, err := parseFilesListConfig(c.FilesToUpload)
```

Comment:
Move parse login after stepconf.parse part.
NOTE: Could check that files exist before creating release.